test-framework/maven/src/main/java/io/quarkus/maven/it/MojoTestBase.java

```diff
@@ -293,7 +293,7 @@ public static void assertThatOutputWorksCorrectly(String logs) {
         assertThat(logs.contains(infoLogLevel)).isTrue();
         Predicate<String> datePattern = Pattern.compile("\\d{4}-\\d{2}-\\d{2}\\s\\d{2}:\\d{2}:\\d{2},\\d{3}\\s").asPredicate();
         assertThat(datePattern.test(logs)).isTrue();
-        assertThat(logs.contains("features: [cdi, resteasy, servlet, undertow-websockets]")).isTrue();
+        assertThat(logs.contains("features: [cdi, resteasy, security, servlet, undertow-websockets]")).isTrue();
```

Comment:
I don't really like that JAX-RS now includes security by default, we should look at fixing this, usually we work around this by adding some kind of SPI module.

Reply:
could you write a bit more on that, or maybe give me an example?

Reply:
Let's look at it when the other issues have been addressed, I am about to board a flight back to Australia so will be out of contact for a while.

Reply:
Just to add some background: I added it to the resteasy extension because before the changes resteasy had RBAC for JAX-RS implemented.